rero_ils/modules/documents/mappings/v7/documents/document-v0.0.1.json

```diff
@@ -90,56 +156,36 @@
                 "properties": {
                   "value": {
                     "type": "text",
-                    "index": false,
```

Comment:
for my information : Why removing `index: false` for all theses keys ? Is it really necessary to index and store them ?

Reply:
Yes, we want to index theses fields with all the languages analyzers and with the default text analyzer.